test/fixtures/election-98f5203139-choctaw-general-2019/blank-p1-metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "ballotStyleId": "3",
   "isTestBallot": false,
-  "pageCount": 2,
+  "pageCount": 3,
```

Comment:
This and the other similar change has no real effect on the change in this PR, it's just a consistency issue.